integration/src/test/java/org/apache/iotdb/db/integration/IoTDBTtlIT.java

```diff
@@ -39,7 +39,7 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 
-@Category({LocalStandaloneTest.class, ClusterTest.class})
+@Category({LocalStandaloneTest.class})
```

Comment:
why delete ClusterTest?

Reply:
The test case testDefaultTTL() (without any test categories) relies on the change of default_ttl configure parameter. The change is not supported yet in ConfigFactory. New configuration parameters support in test will be submitted in other PRs.

Reply:
got it